### PR TITLE
to avoid update cm after created a new cm

### PR DESCRIPTION
**Pull request by huang339**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
to avoid update cm after created cm
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
NONE

```release-note

```

### CODE REVIEW

File: cloud/pkg/devicecontroller/controller/downstream.go [hunk 1]
```diff
@@ -190,6 +190,7 @@ func (dc *DownstreamController) addToConfigMap(device *v1alpha2.Device) {
 				klog.Errorf("Failed to create config map %v in namespace %v, error %v", nodeConfigMap, device.Namespace, err)
 				return
 			}
+			return
```
Comment (Iceber) [inline cloud/pkg/devicecontroller/controller/downstream.go:193]: Need to print error log

Comment (Iceber) [inline cloud/pkg/devicecontroller/controller/downstream.go:193]: First we should use the `apierrors.IsNotFound` to check for `err`
```go
import apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment (fisherxu) [reply cloud/pkg/devicecontroller/controller/downstream.go:193]: +1, we should check whether the err is NotFound type, if not, we should log err and return

**Comment by huang339** [comment]
already update,please check again

**Comment by fisherxu** [comment]
lgtm, and please signoff the commit to fix the failed DCO ci issue. @huang339

**Comment by fisherxu** [comment]
And looks like the email addr in the commit haven't associated with your github account, please add it to your acount, or github can't identify your contributions.

**Comment by huang339** [comment]
I added the email address to my acount

**Pull request closed without merging**